funding/manager.go

```diff
@@ -1286,6 +1287,49 @@ func (f *Manager) advancePendingChannelState(
 			channel.FundingOutpoint, err)
 	}
 
+	if blockchain.IsCoinBaseTx(confChannel.fundingTx) {
+		// If it's a coinbase transaction, we need to wait for it to
+		// mature. We wait out an additional MinAcceptDepth on top of
+		// the coinbase maturity as an extra margin of safety.
+		maturity := f.cfg.Wallet.Cfg.NetParams.CoinbaseMaturity
+		numCoinbaseConfs := uint32(maturity + channel.NumConfsRequired)
```

Comment:
I don't think we need to add it, we just need to wait for cb maturity. The reason we require confirmations in the first place is in case for whatever reason that the chain reorgs the funding tx out of the history. In this scenario, even with a reorg of `NumConfsRequired + 1` it would be impossible to reorg out the funding tx. So it is guaranteed that at maturity the channelpoint is both in the history and spendable.